MyOwnORM/MyOwnORM/CustomDbContextOptionsBuilder.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class CustomDbContextOptionsBuilder 
```

Comment:
class should also be static

Reply:
class can't be static because I used it as a parameter in the 'OnConfiguring' method in 'CustomDbContext'.